src/widget/controlwidgetconnection.h

```diff
@@ -137,5 +138,7 @@ class ControlWidgetPropertyConnection final : public ControlWidgetConnection {
     void slotControlValueChanged(double v) override;
 
   private:
-    QByteArray m_propertyName;
+    QString m_propertyName;
```

Comment:
Both m_propertyName and m_property should be immutable. This requires to extract fetching of QMetaProperty into a utility function.